spotinst/elastigroup_aws_launch_configuration/fields_spotinst_elastigroup_aws_launch_configuration.go

```diff
@@ -52,6 +52,63 @@ func Setup(fieldsMap map[commons.FieldName]*commons.GenericField) {
 		nil,
 	)
 
+	fieldsMap[Images] = commons.NewGenericField(
```

Comment:
Not sure if this is the right types configuration, according to the API service, we should support array of `image` objects, each single `image` object should contain id fields of type string.

Please align it to be able to apply the following configuration:

```
images = [
    image {
      id = ami-12345
   }
   
  image {
    id = ami-67890
  }
]
```

![image](https://user-images.githubusercontent.com/64735199/187655991-b9056b85-3864-4d6c-809b-d806b60d401e.png)